src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -138,7 +139,7 @@ export class MeetingManager implements AudioVideoObserver {
   // Please use `meetingManagerConfig` to use `MeetingManagerConfig` values.
   postLoggerConfig: PostLogConfig | null = null;
 
-  eventReporter: EventReporter;
+  private _eventController: EventController;
```

Comment:
Do we need to keep a reference to event controller at all? @nainkunal933 do you remove this in your PR?

Reply:
I think with Kunal's PR we can remove it